testdata/azurerm/plan.json

```diff
@@ -0,0 +1,1384 @@
+							},
+							"security_rule_access": {
+								"default": "Allow",
+								"description": "Specifies whether network traffic is allowed or denied by default rule."
+							},
+							"security_rule_description": {
+								"default": "Enable SSH inbound traffic.",
+								"description": "A description of the default rule."
+							},
+							"security_rule_destination_address_prefix": {
+								"default": "*",
+								"description": "Lists of destination address prefixes to match in the default rule."
+							},
+							"security_rule_destination_port_range": {
+								"default": "22",
+								"description": "Default rule destination port or range."
+							},
+							"security_rule_direction": {
+								"default": "Inbound",
+								"description": " Specifies if default rule will be evaluated on incoming or outgoing traffic."
+							},
+							"security_rule_name": {
+								"default": "SSH",
+								"description": "The name of the default security rule."
+							},
+							"security_rule_priority": {
+								"default": 1001,
+								"description": "Specifies the priority of the default rule."
+							},
+							"security_rule_protocol": {
+								"default": "Tcp",
+								"description": "Network protocol that default rule applies to."
+							},
+							"security_rule_source_address_prefix": {
+								"default": "*",
+								"description": "Default rule, CIDR or source IP range or * to match any IP."
+							},
+							"security_rule_source_port_range": {
+								"default": "*",
+								"description": "Default rule source port or range."
+							},
+							"sg_extra_tags": {
+								"default": {},
+								"description": "A map of tags to assign to the security_group resources."
+							},
+							"subnet_id": {
+								"description": "The ID of the Subnet where this Network Interface should be located in."
+							},
+							"vm_size": {
+								"default": "Standard_DS1_v2",
+								"description": "Specifies the size of the Virtual Machine."
+							}
+						}
+					}
+				}
+			},
+			"variables": {
+				"azure_client_id": {
+					"description": "Azure client ID to use."
+				},
+				"azure_client_secret": {
+					"description": "Azure client Secret to use."
+				},
+				"azure_env": {
+					"default": "public",
+					"description": "Azure environment to use. Can be either `public`, `usgovernment`, `german` or `china`."
+				},
+				"azure_location": {
+					"default": "West Europe",
+					"description": "Azure location to use."
+				},
+				"azure_subscription_id": {
+					"description": "Azure subscription ID to use."
+				},
+				"azure_tenant_id": {
+					"description": "Azure tenant ID to use."
+				},
+				"customer": {
+					"description": "Cycloid customer name."
+				},
+				"env": {
+					"description": "Cycloid environment name."
+				},
+				"project": {
+					"description": "Cycloid project name."
+				}
+			}
+		}
+	}
+}
```

Comment:
I didn't see any sensitive data, I assume it got cleaned up? Asking just to double check

Reply:
Yes double checked.